app/api/session_types.py

```diff
@@ -27,7 +27,7 @@ class Meta:
 
     id = fields.Str(dump_only=True)
     name = fields.Str(required=True)
-    length = fields.Str()
+    length = fields.Int()
```

Comment:
so, str then ? I'll open an issue about field validation so that it can be tackled once all the apis are setup. Validation will take some time, and FE needs the api